redwood-treehouse/src/engineMain/kotlin/app/cash/redwood/treehouse/RealViewBinder.kt

```diff
@@ -76,6 +76,7 @@ public class RealViewBinder(
         scope.launch(dispatchers.main) {
           if (firstDiff) {
             firstDiff = false
+            view.protocolDisplayRoot.children(0)!!.clear()
```

Comment:
This should be 1, not 0, because it's a tag and not an index. There's a constant for this at `ChildrenDiff.RootChildrenTag` which is used by things like `ProtocolDisplay`.

Reply:
Hmmm looks like we unconditionally return the children from our current implementations of the root view. I would prefer if we change those as well to only return when `RootChildrenTag` is requested and to throw otherwise. That change can be separate, but switching to `RootChildrenTag` in this change can still be done.

Reply:
Changed all of our `ProtocolDisplayRoot` implementations to use `RootChildrenTag`.